docs/installation/config/openzaak_config.rst

```diff
@@ -152,6 +152,25 @@ Open Notificaties
 
 All done!
 
+Register notification channels
+==============================
+
+Before notifications can be sent to ``kanalen`` in Open Notificaties, these ``kanalen``
+must first be registered via Open Zaak.
+
+In Open Zaak, activate the virtual environment:
+
+.. code-block:: bash
+
+    $ . env/bin/activate
```

Comment:
This is not relevant in the Docker containers - there is no virtual environment.